ebpfsvc/rpc_util.cpp

```diff
@@ -4,11 +4,13 @@
 #include <malloc.h>
 #include "rpc_interface_s.c"
 #include "svc_common.h"
+#include <sddl.h>
```

Comment:
Move up before line 5

Reply:
Thanks for adding the comment.  I'd suggest between lines 4 and 5 explicitly including <windows.h> and then <sddl.h> (with the comment in line 7), and moving lines 5-6 down after the <> includes.